debug_utils/sierra-emu/tests/corelib.rs

```
@@ -0,0 +1,241 @@
+use std::{path::Path, sync::Arc, u64};
+
+use cairo_lang_compiler::{
+    db::RootDatabase,
+    diagnostics::DiagnosticsReporter,
+    project::{check_compiler_path, setup_project},
+};
+use cairo_lang_filesystem::{
+    cfg::{Cfg, CfgSet},
+    ids::CrateId,
+};
+use cairo_lang_runner::{casm_run::format_for_panic, RunResultValue};
+use cairo_lang_sierra_generator::replace_ids::replace_sierra_ids_in_program;
+use cairo_lang_starknet::starknet_plugin_suite;
+use cairo_lang_test_plugin::{
+    compile_test_prepared_db,
+    test_config::{PanicExpectation, TestExpectation},
+    test_plugin_suite, TestCompilation, TestsCompilationConfig,
+};
+use common::jitvalue_to_felt;
+use rayon::iter::{IntoParallelIterator, ParallelIterator};
+use sierra_emu::{run_program, EntryPoint, ProgramTrace, Value};
+use tracing::info;
+use tracing_test::traced_test;
+
+mod common;
+
+#[traced_test]
+#[test]
+#[ignore = "sierra-emu is not fully implemented yet"]
+fn test_corelib() {
```

I think we should not ignore the tests.

We can ignore the error in the CI instead, that way we can:
1. Assert that this test works as intented.
2. Track progress on the sierra emu implementation.

---

I would great, but I don't know how to ignore it from the ci

---

I could ignore it and run it separately. That way we can still run make test during the build check workflow and then ignore the result of this test

---

I did that for now